application-collabora-ui/src/main/resources/Collabora/Code/Main.xml

```diff
@@ -65,7 +65,8 @@
   ## Save action is included in Collabora editor.
   &lt;div class="actionMenu"&gt;
     &lt;div&gt;
-      &lt;a href="${xwiki.getURL('Main.WebHome')}" class='homePage'
+      #set ($homePageRef = $services.model.createDocumentReference($fileDoc.wiki, 'Main', 'WebHome'))
```

Comment:
The wiki home page is configurable. See https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-api/src/main/java/org/xwiki/wiki/descriptor/WikiDescriptor.java#L218 . You need to take it from the wiki descriptor (wiki manager script service).